components/QuestChain/QuestChainV1Page/QuestsEditor.tsx

```diff
@@ -47,17 +56,104 @@ export const QuestsEditor: React.FC<{
 
   const existingLength = questChain.quests.length;
 
-  const onAddQuest = async (name: string, description: string) => {
+  const onAddQuest = async (
```

Comment:
Can we make all these changes in the `V2` page? that is `components/QuestChain/QuestChainV2Page/QuestsEditor.tsx`

Reply:
fixed in [c0104f6](https://github.com/quest-chains/dapp/pull/143/commits/c0104f6ae2a04232f0630666ca4411862c4cb61f)